packages/about-you/composables/src/composables/useProduct/index.ts

```diff
@@ -0,0 +1,9 @@
+import { useProductFactory } from '@vue-storefront/factories';
+import { enhanceProduct } from '../../helpers';
+import { UseProduct, BapiProduct } from '../../types';
+
+const useProduct: (cacheId: string) => UseProduct<BapiProduct> = useProductFactory<BapiProduct, any>({
+  productsSearch: enhanceProduct
```

Comment:
i dont think its enhancing anything, juz mapping, maybe its better to call it `mapProductSearch` ?